### DISCUSSION TITLE
How to start the autoortho "RUN" function from a batch file

### DISCUSSION THREAD

**Discussion by geronimopilot** [kubilus1/autoortho#172] [Q&A]
Hello all,

Does anyone know how to automatically "run", (start), autoortho from a batch file on windows, (computer), start-up?

I start everything from a batch file so when I turn on my computer, autoortho, x-plane 12 and ASXP all start automatically. However, unless you click "run" on the autoortho window an error prompts up the monitor and x-plane 12 shuts down. I just don't know where the "run" function resides.

Thanks!
Scott

**Comment by kubilus1** [comment]
Just execute the program with the '-H' flag and it runs in headless mode.

**Comment by geronimopilot** [comment]
Wow thanks for the quick response!

I am not familiar with the '-H' headless mode.... can you provide an
example please? This is my startup batch file which  resides on my desktop:

rem start "" "C:\Program Files\Process Lasso\ProcessLasso.exe"

timeout 120

start "" "C:\Users\Owner\Downloads\autoortho_win\autoortho_win.exe"

timeout 15

start "" "E:\X-Plane 12\X-Plane.exe"

rem timeout 230

start "" "C:\Program Files\Process Lasso\ProcessLasso.exe"

timeout 14

start "" "C:\Program Files (x86)\HiFi\ASXP12\AS_XPL.exe"

Is it the batch file that gets written to execute the headless mode?

Thanks!
Scott

**Reply by kubilus1** [reply]
I don't use Windows or batch files, but after the autoortho_win.exe is where the '-H' should go, most likely.
